src/main/kotlin/no/nav/syfo/personstatus/db/getFromPersonOversiktStatus.kt

```diff
@@ -38,7 +38,15 @@ const val queryHentUbehandledePersonerTilknyttetEnhet = """
                         SELECT *
                         FROM PERSON_OVERSIKT_STATUS
                         WHERE ((tildelt_enhet = ?)
-                        AND (motebehov_ubehandlet = 't' OR oppfolgingsplan_lps_bistand_ubehandlet = 't' OR (dialogmotekandidat = 't' and dialogmotekandidat_generated_at + INTERVAL '7 DAY' < now())));
+                            AND (motebehov_ubehandlet = 't' 
```

Comment:
Ble lettere å lese nå ja 🥇

Reply:
Hva er `t`? 🤷🏼‍♂️

Reply:
https://dev.to/discdiver/the-true-guide-to-true-and-false-in-postgresql-1p69 😃

Reply:
`TRUE` 👍

Reply:
> The key words TRUE and FALSE are the preferred (SQL-compliant) method for writing Boolean constants in SQL queries."
:trollface: 

Jeg skal begynne å bruke `'ye'`